### PR TITLE
Updates VS Toolkit overview

### PR DESCRIPTION
**Pull request by therealjohn**
Updated to reflect the new extension.

### CODE REVIEW

File: msteams-platform/toolkit/visual-studio-overview.md [hunk 1]
```diff
@@ -15,45 +15,37 @@ The Microsoft Teams Toolkit enables you to create custom Teams apps directly wit
...
-1. Check the box labeled **Place solution and project in the same directory**.
-1. A pop-up window labeled **Add Capabilities** will allow you to choose one or more capabilities for your project setup.
-1. Select the **Next** button to complete the configuration process.
-1. A pop-up window labeled **Add Capabilities** will allow you to choose the properties for each selected capability.
-1. Select **Finish** and you will  land on the **Microsoft Teams Toolkit** landing page.
+1. Launch Visual Studio 2019.
+2. Select **Create a new project**.
+3. Search for **Microsoft Teams App** and select **Next**.
+4. You will arrive at the **Configure your new project** screen where you can choose the **Project name**, **Location**, and **Solution name**.
+5. Select the **Next** button to choose a name for the app.
```
Comment (v-rodriguezd) [inline msteams-platform/toolkit/visual-studio-overview.md:40]: 5. Select **Next** to enter a name for the app.

Question: Please clarify the resulting screen post selecting Next. Are you selecting or entering the app name? Please change the process step as per the action.

**Review by v-rodriguezd — CHANGES_REQUESTED** [review]
Reviewed the modified content. Please verify.

**Comment by v-rodriguezd** [comment]
@therealjohn please incorporate the review changes. Let me know in case of any issues.

File: msteams-platform/toolkit/visual-studio-overview.md [hunk 2]
```diff
@@ -15,45 +15,37 @@ The Microsoft Teams Toolkit enables you to create custom Teams apps directly wit
...
-3. If you would like test your app by deploying it from Visual Studio, you'll need to have IIS (Internet Information Services) installed in your development environment. Visual Studio does not include IIS and it isn't included in the default Windows 10, Windows 8, or Windows 7 configuration; however, you can download the latest version from the [Microsoft download center](https://www.microsoft.com/download/details.aspx?id=48264).
-
-![IIS download page view](../assets/images/iis.png)
-
 ## Install the Teams Toolkit
 
-The Microsoft Teams Toolkit for Visual Studio is available for download from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.vsteamstemplate) or directly from the **Extensions** menu within Visual Studio.
+The Microsoft Teams Toolkit for Visual Studio is available for download from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=msft-vsteamstoolkit.vsteamstoolkit) or directly from the **Extensions** menu within Visual Studio.
 
 ## Using the toolkit
```
[the remark this answers is not in the document]
Comment (therealjohn) [reply msteams-platform/toolkit/visual-studio-overview.md:26]: Not sure what change you are suggesting. Can you clarify?

**Comment by v-rodriguezd** [comment]
@therealjohn Please incorporate the review suggestions. Thank you.

File: msteams-platform/toolkit/visual-studio-overview.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (v-rodriguezd) [reply msteams-platform/toolkit/visual-studio-overview.md:26]: @therealjohn please change the title "Using the toolkit" on line 26 to "Use the Toolkit".

**Comment by v-rodriguezd** [comment]
Hi @therealjohn please incorporate the review comments and let me know in case of any issues. Awaiting your changes.

**Comment by therealjohn** [comment]
@v-rodriguezd Done! Please review. Thanks!

File: msteams-platform/toolkit/visual-studio-overview.md [hunk 3]
```diff
@@ -15,45 +15,37 @@ The Microsoft Teams Toolkit enables you to create custom Teams apps directly wit
...
+5. Select the **Next** button to choose a name for the app.
 
 ## Configure your app
 
 At its core, the Teams app embraces three components:
 
   1. The Microsoft Teams client (web, desktop or mobile) where users interact with your app.
-  1. A server that responds to requests for content that will be displayed in Teams, e.g., HTML tab content or a bot adaptive card .
-  1. A Teams app package consists of three files:
+  2. A server that responds to requests for content that will be displayed in Teams, e.g., HTML tab content or a bot adaptive card .
```
[the remark this answers is not in the document]
Comment (v-rodriguezd) [reply msteams-platform/toolkit/visual-studio-overview.md]: @therealjohn please replace 'e.g.,' with 'for example'. Also please remove extra space before the period in 'bot adaptive card.' Please remove 'that will be'.

File: msteams-platform/toolkit/visual-studio-overview.md [hunk 4]
```diff
@@ -71,32 +63,22 @@ When an app is installed, the Teams client parses the manifest file to determine
...
-1. To configure your app, on the **Microsoft Teams Toolkit** landing page, select **Edit app package**.
-1. From the **My Environments** drop-down menu, select **development**.
-1. You will land on the **App details** page where you can edit your app's property fields.
-1. Editing the fields in the App details page updates the contents of the manifest.json file that will ultimately ship as part of the app package. [Learn more](https://aka.ms/teams-toolkit-manifest)
+1. To configure your app, select the **Project > TeamsFx > Configure for SSO...** menu.
 
-## Package your app
-
-Modifying the **app details** page or updating the **manifest**, or **.env** files in your app's  **.publish** folder will automatically generate your **Development.zip** file. The Development.zip file includes three required files — the **manifest.json** and [two icons](../concepts/build-and-test/apps-package.md#app-icons).
+You will be prompted to sign-in to your Microsoft Account that has a M365 tenant.
```
[the remark this answers is not in the document]
Comment (v-rodriguezd) [reply msteams-platform/toolkit/visual-studio-overview.md:68]: @therealjohn this change is not reflected please replace "You will be prompted to sign-in to your Microsoft Account that has a M365 tenant." with "When prompted, sign in to your Microsoft account that has an M365 tenant."

**Comment by v-rodriguezd** [comment]
Please incorporate the suggestions. There are some changes which are not reflecting.

File: msteams-platform/toolkit/visual-studio-overview.md (hunk 1 above)
Comment (v-rodriguezd) [reply msteams-platform/toolkit/visual-studio-overview.md]: Please rephrase step 5 to "Select Next to enter a name for the app."

File: msteams-platform/toolkit/visual-studio-overview.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (v-rodriguezd) [reply msteams-platform/toolkit/visual-studio-overview.md:68]: Change not reflected.

**Comment by v-rodriguezd** [comment]
There are two comments pending, please incorporate these:
Change "You will be prompted to sign-in to your Microsoft Account that has a M365 tenant." to "When prompted, sign in to your Microsoft account that has an M365 tenant."
Please rephrase step 5 under Set up a new Teams project to "Select Next to enter a name for the app."

**Comment by v-rodriguezd** [comment]
@therealjohn There are two comments pending, please incorporate these:
Change "You will be prompted to sign-in to your Microsoft Account that has a M365 tenant." to "When prompted, sign in to your Microsoft account that has an M365 tenant."
Please rephrase step 5 under Set up a new Teams project to "Select Next to enter a name for the app."

**Comment by v-rodriguezd** [comment]
@therealjohn Please incorporate these suggestions:
Rephrase "You will be prompted to sign-in to your Microsoft Account that has a M365 tenant." to "When prompted, sign in to your Microsoft account that has an M365 tenant."
Please rephrase step 5 under Set up a new Teams project to "Select Next to enter a name for the app."

**Comment by v-rodriguezd** [comment]
@therealjohn Please incorporate these pending suggestions:
Rephrase "You will be prompted to sign-in to your Microsoft Account that has a M365 tenant." to "When prompted, sign in to your Microsoft account that has an M365 tenant."
Please rephrase step 5 under Set up a new Teams project to "Select Next to enter a name for the app."

**Comment by v-rodriguezd** [comment]
@therealjohn please incorporate the two pending suggestions as soon as possible. Awaiting your changes.

**Comment by v-rodriguezd** [comment]
Hi @therealjohn Please rephrase step 5 under Set up a new Teams project to "Select Next to enter a name for the app."

File: msteams-platform/toolkit/visual-studio-overview.md (hunk 1 above)
Comment (v-rodriguezd) [reply msteams-platform/toolkit/visual-studio-overview.md]: Please incorporate this suggestion rephrase step 5 to "Select Next to enter a name for the app."

**Review by v-rodriguezd — APPROVED** [review]
Approved

**Pull request merged**